### DISCUSSION TITLE
How do I get the standard library from neovim-flake?

### DISCUSSION THREAD

**Discussion by Nowaaru** [NotAShelf/nvf#218] [Q&A] [answered]
Using home-manager.

Trying to write lua bindings. 
Tried using lib.
Tried using lib.hm.
Tried using nixpkgs.lib.

Not working. I did read the docs. The method for getting the DAG is not the method for getting anything in the standard library.

**Comment by NotAShelf** [comment]
You'd want to access the lib as exposed by this flake. In most cases, it'll be `inputs.neovim-flake.lib.nvim` You can find available attributes in the lib directory (it's usually the same as the file name without the .nix suffix)

Unless I misunderstand what you mean by "standard library"?

**Reply by Nowaaru** [reply, marked as answer]
I was looking at the bindings section under [`Hacking Neovim Flake`](https://notashelf.github.io/neovim-flake/index.xhtml#ch-hacking) and I was under the impression that I was able to use the `mk` functions to create my bindings instead of directly dropping an attrset.

I thought that I was accessing something completely different when I wasn't able to find the functions. Turns out that I was accessing the standard library, not the *extended* standard library. My apologies :sweat:
